infra/cifuzz/filestore/git/__init__.py

```diff
@@ -0,0 +1,155 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module for a git based filestore."""
+
+from distutils.dir_util import copy_tree
```

Comment:
Why do we need this function instead of the more common ones?
Also, maybe import dir_util and use `dir_util.copytree`? I think that more closely follows conventions against importing individual functions and classes and importing modules instead.